### ISSUE TITLE
Possible to add support for test classes?

### ISSUE THREAD

**Issue by bcrandell** [cprieto/tsqlt-teamcity#6]
Hi Cristian.  Would it be possible to add support for running a single test class instead of running all tests?  I have tests I want to run on every build and others I don't want to run on every build, and they're separated into different test classes.  It would be nice if there was a textbox on the setup screen where I could add the test class(es) to be run and then EXEC [tSQLt].[Run] 'testclass' would be run instead of EXEC [tSQLt].[RunAll].

Thanks,
Brad

**Comment by cprieto** [comment]
Yes, that is a nice feature to have. I will add support for it this weekend. It will be a little tricky because you can specify not only the name of the test class but as well a schema to run all the test inside it.

Thanks for suggesting it.

**Comment by faddison** [comment]
Hi, was this every implemented?

**Comment by cprieto** [comment]
Actually yes, but in the last 8 months I changed jobs and move overseas so it is safe to say it is WIP, it is nearly done together with a massive refactor to make it easy to add things. I already got some time from work to finish and submit this and I have to test it in old and recent TC installations but I hope this will happen _before_ January finishes.

**Comment by faddison** [comment]
Hi, did you manage to make any progress on this?
